src/Component/Editor/Constants.js

```diff
@@ -0,0 +1,18 @@
+const modules = {
+  toolbar: [
+    [{ header: [1, 2, false] }],
+    ["bold", "italic", "underline", "strike","blockquote"],  
```

Comment:
you can remove the comma at the end!! as there is no element at the third index.
" [], [], "  --->  " [], [] "